pkgs/development/lua-modules/overrides.nix

```diff
@@ -264,36 +264,51 @@ with super;
     disabled = luaOlder "5.1" || (luaAtLeast "5.4");
   });
 
-  luv = super.luv.override({
-    # Use system libuv instead of building local and statically linking
-    # This is a hacky way to specify -DWITH_SHARED_LIBUV=ON which
-    # is not possible with luarocks and the current luv rockspec
-    # While at it, remove bundled libuv source entirely to be sure.
-    # We may wish to drop bundled lua submodules too...
-    preBuild = ''
-     sed -i 's,\(option(WITH_SHARED_LIBUV.*\)OFF,\1ON,' CMakeLists.txt
-     rm -rf deps/libuv
-    '';
+  luv = pkgs.stdenv.mkDerivation rec {
```

Comment:
This ignores the auto-generated/updated `luv`, so **if** going that way, we should at least remove the automatic version from `maintainers/scripts/luarocks-packages.csv` and `pkgs/development/lua-modules/generated-packages.nix`

Reply:
I sort of agree but my opinion is not too strong either.. @teto what do you think?

Reply:
I agree with @vcunat ; let*s keep the super.luv.override and just modify what is needed. If the problem is to pin a version, you can do that from maintainers/scripts/luarocks-packages.csv

Reply:
@teto my original motivation was not choosing a specific version, but the builder used to build the derivation. I wanted the regular builder to be used - so it'll be detect the project as a cmake project and be built that way. Is there a way to do that out of `super.luv.override`?

As for the `src` attribute changed, the reason was that the `.src.rock` file doesn't unpack nicely as does the GitHub tarball, _with the standard buildier_. Also, I got the suspicion that some files are missing from the `src.rock` in comparison to the Github tarball and these files are needed for a Cmake build.

Reply:
I've reached the conclusion that removing it from the auto generated set is a good idea - because otherwise, even if `super.luv.version` is updated from the maintainers' script, we'll continue to fetch the old sources and use _them_, because they will always be in hydra's cache because of the `sha256`.

Pushed a fix. @teto please run another Darwin build by @GrahamcOfBorg just to make sure :).